playbook/app/pb_kits/playbook/pb_contact/_contact.tsx

```diff
@@ -83,25 +87,27 @@ const Contact = (props: ContactProps): React.ReactElement => {
     >
       <Body
           className="pb_contact_kit"
-          color="light"
+          color={dark ? "light" : "default"} 
```

Comment:
@anthonymig88, just to make sure. Is this right?
I am asking because "light" was the value before the dark prop. Now, if it is dark, it will be light; if it is not, it will default.

Reply:
Thanks for the review and for pointing that out @carloslimasd ! You’re right, and to clarify:

- If dark is true, the color will be set to "light"
- If dark is false, the color will default to "default"

The goal was to ensure that the icon's color remains visually appropriate and consistent with the overall theme (light or dark) of the component. By making this change, we can ensure that the icon color adjusts dynamically!